### PR TITLE
12.0 add res company active

### PR DESCRIPTION
**Pull request by legalsylvain**
Add new trivial module ``res_company_active`` that allow to disable companies.
Checks are realized to avoid to disable current company or companies with users.

thanks for your review.

res_company_form

validation_error

### CODE REVIEW

File: res_company_active/i18n/fr.po [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+#. module: res_company_active
+#: code:addons/res_company_active/models/res_company.py:25
+#: code:addons/res_company_active/models/res_company.py:26
+#, python-format
+msgid "You can not disable the company %s because it is the current company for the following active users:\n"
+"\n"
+" - %s\n"
+"\n"
+" Please change the company of there users, or disable them"
```
Comment (fcayre) [inline res_company_active/i18n/fr.po]: there users => these users

File: res_company_active/models/res_company.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+                    raise ValidationError(_(
+                        "You can not disable the current company."))
+                # check if it it the current company for some users
+                users = ResUsers.search([('company_id', '=', company.id)])
+                if len(users):
+                    raise ValidationError(_(
+                        "You can not disable the company %s because it is the"
+                        " current company for the following active users:\n\n"
+                        " - %s\n\n"
+                        " Please change the company of there users, or disable"
```
Comment (fcayre) [inline res_company_active/models/res_company.py]: there => these

**Review by fcayre — APPROVED** [review]
Looks very good to me. Not tested functionally.

**Review by bealdav — APPROVED** [review]

**Comment by legalsylvain** [comment]
@bealdav & @fcayre. thanks for your quick review.
@fcayre : typo fixed.

**Pull request merged**